src/components/MapSearchBar/MapSearchBar.css

```diff
@@ -0,0 +1,81 @@
+.map-search-panel {
+  position: absolute;
+  right: 0;
+  margin: 20px;
+  z-index: 1000;
+}
+
+.search-icon {
+  height: 25px;
+  position: absolute;
+  padding: 7px 10px;
+  margin: 10px;
+}
+
+.close.icon.clear-search {
+  position: absolute;
+  right: 0;
+  margin: 8px;
+  cursor: pointer;
+}
+
+.close.icon.clear-search:hover {
+  color: #aaa;
+}
+
+.react-autosuggest__input--focused {
+  outline: none;
+}
+
+.react-autosuggest__input {
+  width: 330px;
+  height: 35px;
+  padding: 10px 30px;
+  padding-right: 25px;
+  font-family: Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  border: 1px solid #aaa;
+  border-radius: 4px;
+  z-index: 1000;
+}
+
+.react-autosuggest__input--open {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.react-autosuggest__suggestions-container {
+  display: none;
+}
+
+.react-autosuggest__suggestions-container--open {
+  display: block;
+  position: absolute;
+  top: 51px;
```

Comment:
This might need to be moved up a little to make sure the suggestion list appears right under the input box.